### PR TITLE
Update with the correct docker image integritee-network/deployment#7

### PR DESCRIPTION
**Pull request by mosonyi**
Added new docker image according to
integritee-network/worker#483
integritee-network/deployment#7

### CODE REVIEW

**Review by clangenb — APPROVED** [review]
Looks Good.

👍 for the rustup show. :)

File: src/howto_private_tx.md [hunk 1]
```diff
@@ -16,18 +16,19 @@ Build worker, client and node in our docker:
...
 mkdir demo && cd demo
-docker run -it -v $(pwd):/root/work scssubstratee/substratee_dev:1804-2.12-1.1.3-001 /bin/bash
+docker run -it -v $(pwd):/root/work integritee/integritee-dev:0.1.7 /bin/bash
 
 # now you are inside the container
 # clone and build the worker and the client
 cd work
 git clone https://github.com/integritee-network/worker.git
 cd worker
+rustup show
```
Comment (clangenb) [inline src/howto_private_tx.md:32]: Maybe you want to note here that this command installs to correct rust-toolchain to run the worker.

Comment (mosonyi) [reply src/howto_private_tx.md:32]: Did it, thanks for the tip.

**Review by brenzi — APPROVED** [review]

File: src/howto_private_tx.md (hunk 1 above)
Comment (haerdib) [reply src/howto_private_tx.md:32]: Could you also add this to the howto_direct_tx demo?

Comment (haerdib) [reply src/howto_private_tx.md:32]: And with this the 

```
# initialize wasm build environment
./scripts/init.sh
```
below is obsolete now?

Comment (clangenb) [reply src/howto_private_tx.md:32]: yes, this is actually obsolete in most of our repos.

Comment (mosonyi) [reply src/howto_private_tx.md:32]: Thanks for your feedback, changes are already in for your review again.

**Review by haerdib — APPROVED** [review]
Looks good, thanks!

**Comment by mosonyi** [comment]
@brenzi if you approve, please merge it as I don't have privileges to do that.

**Pull request merged**